tests/samples/hello-numpy/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.14...3.18)
+
+project(hello-pybind11 VERSION "0.1")
+
+# Define CMAKE_INSTALL_xxx: LIBDIR, INCLUDEDIR
+include(GNUInstallDirs)
+
+find_package(Python REQUIRED COMPONENTS Development NumPy)
+
+# Fetch pybind11
+include(FetchContent)
+
+FetchContent_Declare(
+  pybind11
+  URL https://github.com/pybind/pybind11/archive/refs/tags/v2.6.2.tar.gz
```

Comment:
Guessing this (and the ...3.18 above) were copied from another example? We really should be update the pins/limits. Doesn't have to be in this PR, though, just noting for later.